frontend/src/workbench/RightSite.tsx

```diff
@@ -11,93 +11,79 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-import { makeStyles } from '@material-ui/core/styles';
-import Tab from '@material-ui/core/Tab';
-import Tabs from '@material-ui/core/Tabs';
+import MuiAccordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
+import MuiCollapse from '@material-ui/core/Collapse';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
 import { PropertiesWebSocketContainer } from 'properties/PropertiesWebSocketContainer';
-import React, { useState } from 'react';
-import { ValidationWebSocketContainer } from 'validation/ValidationWebSocketContainer';
-import { RightSiteProps, TabPanelProps } from './RightSite.types';
+import React from 'react';
+import { RightSiteProps } from './RightSite.types';
 
 const useSiteStyles = makeStyles((theme) => ({
   site: {
-    display: 'flex',
-    flexDirection: 'column',
-    '& > *:nth-child(2)': {
-      flexGrow: 1,
-    },
-  },
-  tabsRoot: {
-    minHeight: theme.spacing(4),
-    borderBottomColor: theme.palette.divider,
-    borderBottomWidth: '1px',
-    borderBottomStyle: 'solid',
+    display: 'grid',
+    gridTemplateColumns: 'minmax(0,1fr)',
+    gridTemplateRows: 'minmax(0,1fr)',
   },
-  tabRoot: {
-    minHeight: theme.spacing(4),
-    textTransform: 'none',
+  accordionDetailsRoot: {
+    display: 'block',
```

Comment:
I'm curious, why change the regular display policy?

Reply:
If I keep the default behavior, it uses flexbox to display the details view content, and thus, displays a horizontal scrollbar when properties names are too long. With `display: 'block'`, there is no horizontal scrollbar, and when properties names are too long, there are displayed with ellipses.